src/ppx_deriving_yojson.ml

```diff
@@ -492,7 +468,7 @@ let ser_str_of_type_ext ~options ~path:_ ({ ptyext_path = { loc }} as type_ext)
       Ppx_deriving.mangle_lid
         (`PrefixSuffix ("M", "to_yojson")) type_ext.ptyext_path.txt
     in
-    String.concat "." (Longident.flatten mod_lid)
+    String.concat "." (Longident.flatten_exn mod_lid)
```

Comment:
The whole line could be written `Longident.name mod_lid` instead.

Reply:
Done in https://github.com/ocaml-ppx/ppx_deriving_yojson/pull/121/commits/5a2680829a9d1cb30821d296b3ec66cb56fc7cc1